### ISSUE TITLE
Unable to establish connection on any API requests

### ISSUE THREAD

**Issue by rcamps** [merge-api/merge-ruby-client#24]
Version 0.1.1 fails on every API request, including link_token requests. 

<img width="1095" alt="Screenshot 2024-05-24 at 10 28 45 AM" src="https://github.com/merge-api/merge-ruby-client/assets/25013996/db133a11-7b4a-4645-bbfa-17322a7b2bf7">

**Comment by armandobelardo** [comment]
@rcamps would you be able to send over where you instantiate the client? I can't seem to reproduce this on my end

**Comment by rcamps** [comment]
<img width="580" alt="Screenshot 2024-05-27 at 10 48 21 AM" src="https://github.com/merge-api/merge-ruby-client/assets/25013996/d8ab18de-f31c-456c-b20c-d69056c03331">

We are instantiating the client like this - API key being our latest API key we created for this purpose, environment being `Environment::TEST`.

**Comment by armandobelardo** [comment]
@rcamps there is no `Environment::TEST` (https://github.com/merge-api/merge-ruby-client/blob/main/lib/environment.rb), could you update that to a valid environment?
Using `Environment::TEST` would lead to a nil value piping through, which could explain this error you're seeing.

**Comment by rcamps** [comment]
<img width="1430" alt="Screenshot 2024-05-27 at 11 51 32 AM" src="https://github.com/merge-api/merge-ruby-client/assets/25013996/e9f4f3e5-1742-4e5b-8a53-d611b7c27d52">
<img width="1062" alt="Screenshot 2024-05-27 at 11 53 47 AM" src="https://github.com/merge-api/merge-ruby-client/assets/25013996/8ac73574-003b-418e-9f7c-19d8fa227fe5">

Ahh good to know - unfortunately we're still running into issues trying to make API calls with that. I generated a new API key from merge just to rule out any key mismatch, still getting the same error.

**Comment by armandobelardo** [comment]
Hi @rcamps that environment is a Ruby class as linked above, to use it the code would be: `client = Merge::Client.new(environment: Merge::Environment::PRODUCTION, api_key: 'YOUR_KEY')`
So it should not be wrapped in quotes

You can find example usage documented throughout the code in yardoc, if that's helpful!

**Comment by armandobelardo** [comment]
It's also worth noting that environment is an optional field, as outlined within the typehinting/yardoc as well as the README of this repo.

**Comment by rcamps** [comment]
Yeah we've tried to pass this in as noted above, but we can't initialize the client at all that way:
<img width="1432" alt="Screenshot 2024-05-27 at 12 52 26 PM" src="https://github.com/merge-api/merge-ruby-client/assets/25013996/63649b8c-b668-4a4d-b24f-6a08f003baed">

We've also tried initializing without the `environment` param, and we get the same 404 error as with it.

**Comment by rcamps** [comment]
We have tested with all the `Merge::Environment::*` params as well as no `environment` param - we get the same Faraday 404 on all API requests. Only version 0.0.3 works for us currently.

<img width="1427" alt="Screenshot 2024-05-27 at 1 03 29 PM" src="https://github.com/merge-api/merge-ruby-client/assets/25013996/20f10900-3016-44fb-9db4-ad326d4d8e62">

<img width="1433" alt="Screenshot 2024-05-27 at 1 04 32 PM" src="https://github.com/merge-api/merge-ruby-client/assets/25013996/e0faca14-062d-47ef-8c54-2bf6181146d8">

**Comment by armandobelardo** [comment]
A 404 is a different error than the above, I'll dig in to see if I can repro a 404

**Comment by rcamps** [comment]
I think the issue is just the URL being generated - `https://api.merge.dev/ticketing/v1/link-token`

It's just missing a /api, i.e. `https://api.merge.dev/api/ticketing/v1/link-token`

![image](https://github.com/merge-api/merge-ruby-client/assets/25013996/77a28171-4b07-48b4-9986-32aaf5e3d512)
